nuget/spec/dependabot/nuget/nuget_config_credential_helpers_spec.rb

```diff
@@ -32,8 +32,13 @@
     context "with non-empty credential set" do
       let(:credentials) do
         [
-          { "type" => "nuget_feed", "url" => "https://private.nuget.example.com/index.json",
+          # private feed with a token
+          { "type" => "nuget_feed", "url" => "https://private1.nuget.example.com/index.json",
             "token" => "secret_token" },
+          # private feed with a username and password
+          { "type" => "nuget_feed", "url" => "https://private2.nuget.example.com/index.json", "username" => "my",
+            "password" => "passw0rd" },
```

Comment:
Any value in a test with all three set that ensures we take the `token`?

Reply:
Done.